tests/utilities/index.ts

```diff
@@ -99,6 +104,26 @@ export class Workspace {
     );
   }
 
+  async writeFilesFromFixtureDirectory(fixtureDirectoryPath: string) {
+    const fileList = glob('**/*.*', {
```

Comment:
I think this can be:

```suggestion
    const fileList = glob('**/*', {
```

Reply:
We actually need the `*.*` or else we get a directory read error.

Reply:
`ENOTEMPTY: directory not empty, rmdir '/Users/stephenhukish/src/github.com/Shopify/loom/tmp/test-workspace-0.8364581935113262'`

Reply:
Perhaps we need to add a `nodir: true` option to tell it to ignore directories?

Otherwise it seems like we're assuming file names will always have a dot in them (not always true) and directories will never have a dot in them (also not always true)

Reply:
Lets make another PR for this as I want these tests in.

Reply:
nvm I am hangry and this was a super quick change lol